trace/opencensus/trace/ext/utils.py

```diff
@@ -23,3 +29,20 @@ def get_func_name(func):
         return '{}.{}'.format(module_name, func_name)
 
     return func_name
+
+
+def disable_tracing_url(url, blacklist_paths=None):
+    """Disable tracing on the provided blacklist paths, by default not tracing
+    the health check request. True if not tracing, False if tracing.
+
+    :type blacklist_paths: list
+    :param blacklist_paths: Paths that not tracing.
+    """
+    if blacklist_paths is None:
+        blacklist_paths = DEFAULT_BLACKLIST_PATHS
+
+    for path in blacklist_paths:
+        if path in url:
```

Comment:
I worry a bit about false positives here, like the url is `foo.com/something?blahblahblah&arg=_/ah/health`.  What would you think about requiring the path to be at the start of the URL, or be a regular expression that matches the entire URL?  You'd have to do a little more work to split the hostname/port from the path from the fragment/query parameters.

Reply:
Done. Split the url path from the provided url, and see whether the url path starts with the blacklisted path or not.